packages/gafl-webapp-service/src/routes/journey-definition.js

```diff
@@ -257,7 +258,15 @@ export default [
         page: CONTACT_SUMMARY
       }
     },
-    backLink: s => (s.fromSummary === CONTACT_SUMMARY_SEEN ? CONTACT_SUMMARY.uri : ADDRESS_LOOKUP.uri)
+    backLink: (s, t) => {
```

Comment:
I know this isn't all your doing, but please could you rename `s` and `t` to something a bit more meaningful?

Reply:
Yes will do, I'll rename to status and transaction.